web/src/views/ErrorPage.tsx

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,git
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+interface Props {
+    errorMessage: string
+}
+
+const ErrorPage: React.FC<Props> = ({errorMessage}) => {
+  return (
+    <div>
+      <h1>React Error</h1>
+      Something seems to have gone wrong...
+      <div id="errorPageMessage">
```

Comment:
We  use rebass across the project, try to follow the pattern

Reply:
Please be more specific. I have no idea what we're actually trying to address here. I used NotFound.tsx as an example for this view as I figured they were reasonably close in terms of function.